sirepo/pkcli/test_http.py

```diff
@@ -190,15 +192,19 @@ async def get_sim(self, sim_name):
 
     def pkdebug_str(self):
         return pkdformat(
-            '{}(sim_type={} email={})',
+            '{}(sim_type={} email={} uid={})',
             self.__class__.__name__,
             self.sim_type,
             self.client.email,
+            self.client.uid,
         )
 
 
 async def _cancel_all_tasks(tasks):
     for t in tasks:
+        # Sleep a bit to not completely flood server
```

Comment:
document this as, say, `await pause_for_server()`

Reply:
https://github.com/radiasoft/sirepo/pull/3560/commits/f1f9b1aa4be4c77396e3467d5038c7b700e3997a